public/docs/_examples/dependency-injection/dart/test/hero_list_component_test.dart

```diff
@@ -0,0 +1,36 @@
+// Simulate a simple test
+
+// Reader should look to the testing chapter for the real thing
```

Comment:
This is a real test, not a simulation, right? So how about these comments instead:

```
// A simple test
// More details will be in the testing chapter.
```